src/AutomatedCar/SystemComponents/Packets/PedalPacket.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace AutomatedCar.SystemComponents.Packets
+{
+    using ReactiveUI;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public class PedalPacket : ReactiveObject
```

Comment:
Maybe we can use one packet for all user input, kind of a UserInput or HmiInput then later on you can add here the gearShiftUp/DownRequest, handbreak request etc.